packages/angular_devkit/build_angular/src/utils/index-file/augment-index-html.ts

```diff
@@ -214,7 +234,13 @@ export async function augmentIndexHtml(
           for (const linkTag of linkTags) {
             rewriter.emitRaw(linkTag);
           }
-
+          if (imageDomains) {
+            for (const imageDomain of imageDomains) {
+              if (!foundPreconnects.has(imageDomain)) {
```

Comment:
Oh, maybe I misunderstood. You're saying that they've already got a crossorigin preconnect to another domain (because they're going to fetch a script or something), but they're using that same domain to load images. So in that case there would need to be both a crossorigin and a non-crossorigin preconnect link for the same domain?

Reply:
Yes, that was my concern. Although in the image domain context this is probably more of an edge case.  Maybe a comment regarding the situation would be sufficient here.  The crossorigin attribute is one of the more confusing attributes for link elements.
If we implemented generic deduplication in the future, we can factor crossorigin into the logic at that time.